scripts/gui/CourseDisplay.lua

```diff
@@ -170,6 +170,10 @@ function SimpleCourseDisplay:init()
 end
 
 function SimpleCourseDisplay:setVisibility(visible)
+	if not self.rootNode then

```

Comment:
I'd prefer something like we do elsewhere:
`if self.rootNode and entityExists(self.rootNode) then .. `

That way there is no need to do anything elsewhere.